backend/blueprint/api/run.py

```diff
@@ -30,6 +20,7 @@
 app.register_blueprint(homePage.router)
 app.register_blueprint(errorHandlers.router)
 
+
```

Comment:
No need of this as the previous space was fine according to hound configurations :+1:

Reply:
I ran autopep8 over whole codebase